tools/list-repo-maintenance-commits.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
```

Comment:
Turns out my shell wasn't using the latest version of bash installed on my machine. This was what was causing the errors earlier. When I explicitly enter a new shell with `bash`, I get no errors.

Would it make sense to change this to

```suggestion
#!/usr/bin/env bash
```

In my test, it helps my machine pick up the right version of Bash in my PATH.

Reply:
Yeah, and we probably should also include the upgrade check too.

Reply:
312f982c52